### ISSUE TITLE
Two full TS servers spawned for on TS 3.9.x

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#104456]
Issue Type: <b>Bug</b>

**repo**
1. `npm i -g typescript@3.9`
1. Set your user tsdk to use this version, for example:

    ```
    	"typescript.tsdk": "/Users/matb/.nvm/versions/node/v12.14.0/lib/node_modules/typescript/lib"
    ```

1. Now load the VS Code workspace (making sure you are using your user TS version)
1. Open a file in `src` and try getting the document outline

**Bug**
Big delay. 

This is because two full TS Servers are being spawned instead of a syntax only server + full server (see memory usage in screenshot):

<img width="1079" alt="Screen Shot 2020-08-11 at 3 13 33 PM" src="https://user-images.githubusercontent.com/12821956/89954690-6dd72a00-dbe6-11ea-88e7-1116933c1074.png">

VS Code version: Code - Insiders 1.48.0-insider (4c23fc22ced6d0c1f58215ce91ccd08d0c8a0006, 2020-08-11T10:13:04.551Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|6, 6, 5|
|Memory (System)|32.00GB (3.02GB free)|
|Process Argv|-psn_0_42174518|
|Screen Reader|no|
|VM|14%|
</details>

**Comment by mjbvz** [comment]
Seems like this was caused by #104123

**Comment by mjbvz** [comment]
For verification, try the following cases:

- Using TS 3.9.7
- Using TS 4.0.1-rc
- Using TS@next
- Using an old TS version like 3.0

All should load the document symbols quickly even while the loading spinner is showing

**Comment by isidorn** [comment]
Assigning this to july milestone since it is a candidate.

**Comment by isidorn** [comment]
@mjbvz following your steps I have verified that the outline get loaded quickly, however I still see two tsservers spinning up when I open the process explorer. If I understood correctly this is by design, since only one of them has a large memory usage.
I have verified for TS: 3.9.7, 4.0.1-rc. ts@next, 3.5

![Screenshot 2020-08-13 at 12 30 55](https://user-images.githubusercontent.com/1926584/90124619-fc6fb680-dd60-11ea-9017-4ad254d9699d.png)
